proxy/core/src/main/scala/io/cloudstate/proxy/EntityDiscoveryManager.scala

```diff
@@ -140,7 +140,10 @@ class EntityDiscoveryManager(config: EntityDiscoveryManager.Configuration)(
   import EntityDiscoveryManager.Ready
 
   private[this] final val clientSettings =
-    GrpcClientSettings.connectToServiceAt(config.userFunctionInterface, config.userFunctionPort).withTls(false)
+    GrpcClientSettings
+      .connectToServiceAt(config.userFunctionInterface, config.userFunctionPort)
+      .withChannelBuilderOverrides(_.maxInboundMessageSize((4 * 1e6).toInt))
```

Comment:
Can we make this configurable in the proxy config?

Reply:
Yea I will make that modification.